recipes/jhbuild/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "jhbuild" %}
+{% set version = "3.38.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.gnome.org/GNOME/jhbuild/-/archive/{{ version }}/jhbuild-{{ version }}.tar.gz
+  sha256: 5756f50154435c533419acd84671226d1d3cf9f1f6add84e3684340fa7a9e9f1
+
+build:
+  noarch: python
+  number: 0
+  script: |
+    aclocal --install
+    autoreconf --install
+    ./configure --prefix=$PREFIX
+    make
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - gettext
+    - pkgconfig
+    - make
```

Seems to require a C compiler
```suggestion
    - make
    - {{ compiler('c') }}
```